src/lib/api/people.ts

```diff
@@ -0,0 +1,21 @@
+import { Person } from '../../types/Person';
+
+const wait = (delay: number): Promise<typeof delay> => {
+  return new Promise((resolve) => {
+    setTimeout(() => resolve(delay), delay);
+  });
+};
+
+export const getPeople = async (): Promise<Person[]> => {
+  await wait(1000);
```

Comment:
Why did you use timeout here? For spinner?

Reply:
Exactly! So that everyone can see it:)